tests/testthat/test-timer.R

```diff
@@ -78,10 +78,10 @@ test_that("getDomainTimeMs works", {
   t3 <- getDomainTimeMs(list(.now = function(){456}))
   end <- Sys.time()
```

Comment:
I think it would be clearer with:

```R
as.numeric(Sys.time()) * 1000
```

The `as.numeric` here makes it clear what the value is that will be compared later -- this way, `start`, `end`, `t1`, etc. all have the same type.

Reply:
Sorry, I wasn't clear: I to store all the times as a number, as in `start <- as.numeric(Sys.time()) * 1000`